### PR TITLE
updated_text

### LINKED ISSUE #26896: ja: Make docs/concepts/workloads/controllers/_index.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/concepts/workloads/controllers/_index.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/concepts/workloads/controllers/_index.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/workloads/controllers/_index.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/workloads/controllers/_index.md

diff between translated and v1.19
https://gist.github.com/421c7c642399ce6bbde938feb84d35fc

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by himanshu007-creator**
closes #26896
This makes <code>docs/concepts/workloads/controllers/_index.md </code> to follow v1.19 of the original (English) text.

### CODE REVIEW

**Comment by himanshu007-creator** [comment]
I signed it

**Comment by himanshu007-creator** [comment]
"I signed it"

File: content/ja/docs/concepts/workloads/controllers/_index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
-title: "コントローラー"
+title: "Workload Resources"
```
Comment (atoato88) [inline content/ja/docs/concepts/workloads/controllers/_index.md:2]:
```suggestion
title: "ワークロードリソース"
```
Here is the place for Japanese translation.
So I think that it's good to write in Japanese.

**Comment by atoato88** [comment]
@himanshu007-creator 
How about my comment above?
Could you update this PR if you OK :smile:

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator
> How about my comment above?
> Could you update this PR if you OK 😄

Will do'👍🏽

File: content/ja/docs/concepts/workloads/controllers/_index.md (hunk 1 above)
Comment (himanshu007-creator) [reply content/ja/docs/concepts/workloads/controllers/_index.md]: done

**Comment by atoato88** [comment]
/lgtm
Thank you to update! :smile: 

You should assign approver for smooth PR merging as commented by k8s-ci-robot [here](https://github.com/kubernetes/website/pull/27256#issuecomment-808769463)

**Pull request merged**